e2e/e2e.js

```diff
@@ -29,15 +29,67 @@ test('sdk init test', async () => {
 });
```

Comment:
should add .env support for easier testing: https://github.com/adobe/aio-lib-campaign-standard/blob/003f4974ec8dbddb7f2738faece2d2b47c639ac1/e2e/e2e.js#L18